apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2

```diff
@@ -45,6 +45,17 @@ spec:
               value: DekiScript:Web
             - name: server__autorequire__Wiki
               value: DekiScript:Wiki
+            - name: NEW_RELIC_APP_NAME
+              value: {{ NEW_RELIC_APP_NAME }}
+            - name: NEW_RELIC_CONFIG_FILE
+              value: {{ NEW_RELIC_CONFIG_FILE }}
+            - name: NEW_RELIC_MONITOR_MODE
+              value: "{{ NEW_RELIC_MONITOR_MODE }}"
+            - name: NEW_RELIC_LICENSE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: newrelic-secrets
+                  key: NEW_RELIC_LICENSE_KEY
```

Comment:
For kumascript, we'll need to set the "always" config (high security mode, logging, etc) in Kumascript. PR incoming.

We need to change the env variables to the underscore notation:

* ``new relic__license_key`` = ``NEW_RELIC_LICENSE_KEY`` secret
* ``new_relic__app_name`` = ``{{ NEW_RELIC_APP_NAME }}`` - This may need to be formatted as a JSON list if there are more than one.
* ``new_relic__agent_enabled`` = ``{{ NEW_RELIC_MONITOR_MODE }}``

We don't need ``{{ NEW_RELIC_CONFIG_FILE }}`` for KumaScript.

Reply:
note:
`key: NEW_RELIC_LICENSE_KEY` is the key used to pull a value from secrets, but the `- name: NEW_RELIC_LICENSE_KEY` can be changed to whatever the app needs.

Reply:
I'm still working on this, but I'm probably wrong about the underscore notation. I will test locally before submitting the PR.

Reply:
OK, it's much easier than I thought. No Kumascript code changes needed, just environment variables.

The [default kumascript config for New Relic](https://github.com/mdn/kumascript/blob/3f13ac5b2c7890a0c502510cd1b878d7334fc5a1/lib/kumascript/conf.js#L23-L30) already has most of the settings we need.  We could override these, by defining environment variables like ``new_relic__license_key``.

However, the [New Relic Node Agent docs](https://docs.newrelic.com/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration) mention override environment variables.  I've tested these in the Docker dev environment, and I prefer them  to the underscore overrides.

So, keep these like you have them already in this PR:
* ``NEW_RELIC_APP_NAME``
* ``NEW_RELIC_LICENSE_KEY``

``NEW_RELIC_MONITOR_MODE`` needs to be renamed ``NEW_RELIC_ENABLED`` for kumascript, but can be sourced from ``{{NEW_RELIC_MONITOR_MODE}}`` as well.

``NEW_RELIC_CONFIG_FILE`` is not needed in KumaScript.

The new hard-coded environment variables needed are:
* ``NEW_RELIC_LOG=stderr``
* ``NEW_RELIC_BROWSER_MONITOR_ENABLE=false``

When we're 100% out of SCL3, we can make these the default in the KumaScript config, if we want.

Reply:
@jwhitlock do `NEW_RELIC_LOG` and `NEW_RELIC_BROWSER_MONITOR_ENABLE` get set for all deployments?

Reply:
update: I added `NEW_RELIC_LOG` and `NEW_RELIC_BROWSER_MONITOR_ENABLE` for each deployment.

Reply:
Thanks. They are set to those values for all deployments. After SCL3, we can move them to the default config and out of terraform, if we want to tidy up.